packages/geoprocessing/src/toolbox/overlapFeatures.ts

```diff
@@ -163,43 +190,41 @@ const getSketchPolygonIntersectArea = (
       return rem;
     })
     .reduce((sumSoFar, rem) => (rem ? area(rem) + sumSoFar : sumSoFar), 0);
-  return sketchValue;
+  return { value: sketchValue, indices: [] };
 };
 
 /**
  * Sums the value of intersecting features.  No support for partial, counts the whole feature
- * @param featureA
- * @param featuresB
- * @param sumProperty
- * @returns
+ * @param featureA Single sketch (as a feature)
+ * @param featuresB Features to overlap with sketch
+ * @param sumProperty Property with value to sum, if not defined each feature will count as 1
+ * @returns Sum of features/feature property which overlap with the sketch, and a list of
+ * indices for features that overlap with the sketch to be used in calculating total sum of
+ * the sketch collection
  */
 const getSketchPolygonIntersectSumValue = (
   featureA: Feature<Polygon | MultiPolygon>,
   featuresB: Feature<Polygon | MultiPolygon>[],
-  /** Property with value to sum, if not defined each feature will count as 1 */
   sumProperty?: string
 ) => {
+  let indices: number[] = [];
   // intersect and get sum of remainder
   const sketchValue = featuresB
-    .map((curFeature) => {
+    .map((curFeature, index) => {
       const rem = clip(
         featureCollection([featureA, curFeature]),
         "intersection"
       );
-      let count: number = 0;
-      if (!rem) {
-        count = 0;
-      } else if (!sumProperty) {
-        count = 1;
-      } else if (curFeature.properties![sumProperty] >= 0) {
-        count = curFeature.properties![sumProperty];
-      } else {
-        count = 1;
-      }
-      return {
-        count,
-      };
+
+      if (!rem) return { count: 0 };
+
+      indices.push(index);
+
+      if (!sumProperty) return { count: 1 };
+      else if (curFeature.properties![sumProperty] >= 0)
```

Comment:
It seems these first two conditions could be condensed into one condition
`if (sumProperty && curFeature.properties![sumProperty] >= 0)`

The !sumProperty case would just fall through to the return of count 1